config/app.rb

```diff
@@ -0,0 +1,3 @@
+CONFIG = {
+  dimensions: [:searched_repo, :searched_from]
```

Comment:
This is just a stub of a `CONFIG`. Also not sure if this is the best config pattern to be using (and we should modify it to allow for different envs).